app/client/components/user/SingleUserManagement.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import { editUser, resetPin } from "../../reducers/userReducer";
+import { TransferUsage } from "../../reducers/transferUsage/types";
+import { ReduxState } from "../../reducers/rootReducer";
+import EditUserForm, { ICreateUserUpdate } from "./EditUserForm";
+
+interface DispatchProps {
+  editUser: (body: any, path: any) => void;
+  resetPin: (body: any) => void;
+}
+
+interface StateProps {
+  users: any;
+  selectedUser: any;
+  transferUsages: TransferUsage[];
+}
+
+interface OuterProps {
+  userId: number;
+}
+
+type Form = ICreateUserUpdate;
```

Comment:
`Form` is an identical alias of `ICreateUserUpdate` which is an identical alias of `IEditUser`... can we simplify this a bit?

Reply:
Updated